webpack.prod.js

```diff
@@ -0,0 +1,34 @@
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+
+module.exports = merge(common, {
+  mode: 'production',
+  devtool: 'source-map',
+  optimization: {
+    minimizer: [
+      new UglifyJsPlugin({
+        cache: true,
+        parallel: true,
+        sourceMap: true
+      }),
+      new OptimizeCSSAssetsPlugin({})
+    ]
+  },
+  plugins: [
+    new MiniCssExtractPlugin({
+      filename: '[name].css',
+      chunkFilename: '[id].css'
+    })
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: [MiniCssExtractPlugin.loader, 'css-loader']
```

Comment:
This generally looks good, however when I run dist against a simple web server I get the following.

![screen shot 2018-11-28 at 1 50 33 pm](https://user-images.githubusercontent.com/5942899/49174751-06475100-f315-11e8-9c34-430c2bd653b5.png)

Markup and JS seem to be working fine, but the styles are not being applied. I do get a 200 response for the stylesheet, but no styles in the visual sense. Do you see the same on your end?

Reply:
Yep I see the same thing. Looking into it now.

Reply:
This might be why:
"
It's because styled-components injects styles through the CSSOM API, which the Dev Tools in Chrome (and every other browser AFAIK) can't handle. See this ticket: https://bugs.chromium.org/p/chromium/issues/detail?id=387952

Note that this is only true when styled-components is in production mode, i.e. process.env.NODE_ENV is set to "production". As long as you aren't in production mode, styled-components should generate normal <style> tags, which you can interact with through the Dev Tools.

"

Reply:
It looks like the issue stems from react-core having side-effects: false in its package.json. The extract plugin does not include the referenced css in the css bundle as nothing from the base.css is explicitly used when pulling it in. 

https://github.com/webpack-contrib/mini-css-extract-plugin/issues/102#issuecomment-384947142